doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+- Change Over Time 
+- Comparison 
+- Correlation 
+- Distribution 
+- Geospatial 
+- Overview + Detail 
+- Part to Whole 
+- Ranking 
+
+Power your most demanding visualizations with Infragistics $PlatformShort$ charting! 
+
+## $PlatformShort$ Chart Example
+
+This $PlatformShort$ column chart, or column graph, example shows Olympics medals by Country. The $PlatformShort$ column chart belongs to a group of category charts that include at one or more numeric or time-series based data point, and one string data point, that compares data values by differing column heights. In this example, the Y-Axis, or left-side labels of the chart, are displaying numeric data showing the number of medals won, and the X-Axis, or bottom labels, are showing a time value, in this case Year. The columns are the category values representing different countries, like USA, Japan and Germany. 
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Chart and Graph Types
+
+The $PlatformShort$ product has over 65 different chart and graph types for any scenario – from a single chart display to an interactive dashboard. You can create $PlatformShort$ charts like Pie, Bar, Area, Line, Point, Stacked, Doughnut, Scatter, Gauge, Polar, Treemap, Stock, Financial, Geospatial Maps and more for your mobile or web apps. The benefit of our $PlatformShort$ chart vs. others is full support for features like: 
+
+- Responsive Web Design built in 
+- Interactive Panning and Zooming with Mouse, Keyboard and Touch 
+- Full Control of Chart Animation 
+- Chart Drill-Down Events 
+- Real-Time Streaming Support 
+- High-Volume (Millions of Data Points) Support 
+- Trends Lines and other Data Analysis features 
+
+Built with a modular design of axis, markers, series, legend, and annotation layers, the $PlatformShort$ chart makes it easy to design a render any type of data story. Build a simple chart with a single data series, or build more complex data stories with multiple series of data, with multiple axis in composite views. 
+
+### $PlatformShort$ Bar Chart
+
+The $PlatformShort$ Bar Chart, or Bar Graph is among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by horizontal bars of equal width and differing lengths. They are ideal for showing variations in the value of an item over time, data distribution, sorted data ranking (high to low, worst to best). Data is represented using a collection of rectangles that extend from the left to right of the chart towards the values of data points.
+
+Learn more about our [bar charts](chart-types-bar.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
Let's see if this type of hand off is ok for everyone. I am sending a link that has the new chart sample for Inspect and also the data for it.

https://app.abstract.com/share/6692a3ab-c75c-41c9-b937-1012299207d9?mode=build&sha=f8f77e96d2e1b324ec0340dec4c32d7fd20f858e

Data:
export class SampleCategoryData {

    public static create(): any[] {
        const data: any[] = [];
        // total olympic medals for top countries
        data.push({"Amazon": "63%"});
        data.push({"Search engines": "48%"});
        data.push({"Retailer sites": "33%"});
        data.push({"Other marketplaces": "25%"});
        data.push({"At the website of the brand": "21%"});
        data.push({"Comparison sites": "10%"});
        data.push({"Social media": "8%"});
        data.push({"Other": "2%"});
        return data;
    }
}

Reply:
BTW I am not 100% sure about the data if it's correct. Excuse me if it's not.

Reply:
Maybe it will be better to use one of these samples, since they show off better the Bar chart:
https://share.goabstract.com/71617b69-7280-4f84-9b09-690bb4d52aa5?mode=build&sha=0d3fe9819950a2ada8692d1e7ee47b3745b940a9

https://share.goabstract.com/c4493019-0756-4b2d-b5c2-a4d69b33c7b2?mode=build&sha=f6a8b92c3e742deaaf93fa5ea8141247a034c197

Reply:
As long as the one we choose has the richness of interactions of the movies example, I am fine with both datasets.

Reply:
Yes, it does have the same interactions:

https://share.goabstract.com/b508a049-f551-4fae-be19-0424a675870e?mode=build&sha=cbb507583316514fd9d47f72830350341a36dbc0

Reply:
The movies data:

[statistic_id317408_highest-grossing-film-franchises-and-series-2020.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5887221/statistic_id317408_highest-grossing-film-franchises-and-series-2020.xlsx)

Reply:
@HUSSAR-mtrela Our idea here, being an Overview topic, is to show more complex and interesting examples. The heading title is Bar chart, so in SEO perspective this should be ok.